applications/FluidDynamicsApplication/python_scripts/adjoint_fluid_analysis.py

```diff
@@ -3,8 +3,8 @@
 import KratosMultiphysics as Kratos
 import KratosMultiphysics.FluidDynamicsApplication as KFluid
 
-from analysis_stage import AnalysisStage
-from fluid_dynamics_analysis import FluidDynamicsAnalysis
+from KratosMultiphysics.analysis_stage import AnalysisStage
+from KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis import FluidDynamicsAnalysis
```

Comment:
I would say that we make separate PRs for this, this will reduce the scope of this PR very much
Also I started doing this / tell ppl to do this for a long time already

New apps already have this, and also many old ones I have ported already (e.g. most of StructuralMechanics and ContactStructuralMechanics FYI @loumalouomega)

this should make your life easier @roigcarlo

Reply:
Can I update my app with this format and it will work with the current system?, OK I will do a PR

Reply:
yes!
In fact I did most of it already some time ago, e.g. here:
https://github.com/KratosMultiphysics/Kratos/blob/7a12b5ab2f5e3eb8840dafa83ee244273f615429/applications/ContactStructuralMechanicsApplication/python_scripts/contact_structural_mechanics_static_solver.py#L8-L12

Reply:
:+1: In fact the problem is that half of the apps use the new system and half use the older. Structural was very easy to deal with for example but fluid was a pain.

Also, a little bit out of topic. I see a huge amount of imports in the middle of the code. This causes a lot of errors and it should only be used in case there is no other choice.